### PR TITLE
227 Unit Tests

### PR DESCRIPTION
**Pull request by ginaxu1**
## Summary
Clean up unit tests in each Go service to use local SQLite db (not Postgresql!) and delete any redundant tests that were actually integration tests (will be added to `tests/integration` in future PR)

## Type of Change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [x] Refactoring (no functional changes)
- [ ] Performance improvement
- [ ] Other (please describe):

## Testing
- [x] I have tested this change locally
- [x] I have added unit tests for new functionality
- [ ] I have tested edge cases
- [x] All existing tests pass

### CODE REVIEW

File: audit-service/services/data_exchange_event.go [hunk 1]
```diff
@@ -37,6 +38,7 @@ func (s *DataExchangeEventService) CreateDataExchangeEvent(ctx context.Context,
 
 	// Create the event record
 	event := &models.DataExchangeEvent{
+		ID:                uuid.New().String(),
```
Comment (sthanikan2000) [inline audit-service/services/data_exchange_event.go]:
```suggestion
```

we should configure this in the real db to generate automatically

**Review by sthanikan2000 — CHANGES_REQUESTED** [review]

**Review by sthanikan2000 — CHANGES_REQUESTED** [review]
There are some issues in the `Orchestration Engine` unit tests. I had fixed them here at #365 
Also I did the following as well:
- Removed the Legacy Files in the `Portal Backend`
- Formatted the codes except the `consent-engine` according to the `lint-check`

Test Coverages:
image
